cypress/e2e/makeAnApplication.cy.ts

```diff
@@ -0,0 +1,56 @@
+import { faker } from '@faker-js/faker/locale/en_GB';
+
+import ApplyPage from '../pages/apply';
+import HomePage from '../pages/home';
+import SignInPage from '../pages/signIn';
+import StartPage from '../pages/start';
+import { screenPresets } from '../support/helpers';
+import {
+  interceptApplicatonApi,
+  interceptAuthApi,
+} from '../support/intercepts';
+
+describe('Housing application', () => {
+  screenPresets.forEach((screenPreset) => {
+    it(`user successfully submits an application on ${screenPreset}`, () => {
```

Comment:
Is this just starting an application at this stage rather than submitting it? Submit would happen once they have completed the form.

Reply:
Yeah good point - will adjust to reflect it's the verification steps.